app/components/markdown/at_mention/at_mention.tsx

```diff
@@ -172,6 +221,11 @@ const AtMention = ({
         mention = displayUsername(user, user.locale, teammateNameDisplay);
         isMention = true;
         canPress = true;
+    } else if (group?.name) {
+        // highlighted = group.isMember(currentUserId);
```

Comment:
Is this comment relevant here?

Reply:
No, will remove - it was a placeholder but highlighting is in the next PR.